lib/ibm_watson/visual_recognition_v3.rb

```diff
@@ -167,13 +162,13 @@ def classify(images_file: nil, accept_language: nil, url: nil, threshold: nil, o
     #########################
 
     ##
-    # @!method detect_faces(images_file: nil, url: nil, images_file_content_type: nil, images_filename: nil)
+    # @!method detect_faces(images_file: nil, url: nil, images_file_content_type: nil, accept_language: nil, images_filename: nil)
```

Comment:
Is this a breaking change? Should `accept_language` be moved after `images_file`?

Reply:
Shouldn't be a problem in Ruby, but I will re-arrange to be consistent with other SDKs.